cmake/cmkr.cmake

```diff
@@ -0,0 +1,253 @@
+include_guard()
```

Comment:
sucks, but cmkr explicitly says that the CMakeFiles.txt is intended to be included in the repo